### ISSUE TITLE
Consider using Greenkeeper

### ISSUE THREAD

**Issue by donmccurdy** [aframevr/aframe#3015]
Various warnings when running `npm install`:

```
npm WARN deprecated uglifyjs@2.4.11: uglifyjs is deprecated - use uglify-js instead.
npm WARN deprecated wrench@1.5.8: wrench.js is deprecated! You should check out fs-extra (https://github.com/jprichardson/node-fs-extra) for any operations you were using wrench for. Thanks for all the usage over the years.
npm WARN deprecated node-uuid@1.4.8: Use uuid module instead
npm WARN deprecated native-or-bluebird@1.2.0: 'native-or-bluebird' is deprecated. Please use 'any-promise' instead.
npm WARN deprecated line-numbers@0.2.0: Copy its ~20 LOC directly into your code instead.
npm WARN deprecated sockjs-client-node@0.1.1: deprecated in favor of the official sockjs-client
npm WARN deprecated graceful-fs@1.2.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
npm WARN deprecated graceful-fs@2.0.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
```

It's a pain to manually deal with those... maybe we could set up something like https://greenkeeper.io/? It should automatically open PRs to update dependencies (triggering Travis CI in the process). Anyone have experience with that, and know of pros/cons?

**Comment by ngokevin** [comment]
I usually prefer to keep them locked because updating dependencies introduces risk. There've been times we will mass-update deps and small API changes can possibly introduce bugs. The warnings should be benign, is there any action needed?

**Comment by dmarcos** [comment]
Agreed. I prefer to keep control of the dependencies. Updating each of them should be a deliberate decision.

**Comment by donmccurdy** [comment]
The graceful-fs example should be fixed before 0.7.0; you cannot build A-Frame with the most recent two versions of Node.js — note that https://nodejs.org/en/ lists v8.4 as the "Current" version of Node.js, and it will become LTS quite soon.

I think the goal of Greenkeeper is that updating dependencies is still a deliberate decision, just with less manual checking... it will verify `npm install && npm test`, open a PR ([example](https://github.com/hoodiehq/hoodie/pull/797)), show you the relevant release notes, and then you decide whether to merge it or pin the version. It would also send an alert if an update to one of your dependencies breaks your build, as happened in #2895, but that could be just as easily handled by npm package locking.

Anyway I don't feel too strongly, except that we do need to update graceful-fs before 0.7.0
